src/MonthView/phone/DayCalendarBox/IndividualComponents/BusyIndicator.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react"
+import { View } from "react-native"
+import styles from "./BusyIndicator.style"
+
+export interface Indicator {
+    name: string
+    color: string
+}
+
+export interface BusyIndicatorProps {
```

Comment:
sorry not nitpicking

can we rename
busyIndicatorHeight => 'height'
Indicator => 'BusyIndicator' so we have consistency with your variable name. Otherwise it can be any indicator

Reply:
changed to BusyIndicatorObj as the object that we are expecting from parent component 👍